rust/noosphere-storage/Cargo.toml

```diff
@@ -54,3 +55,7 @@ features = [
   "Window",
   "DedicatedWorkerGlobalScope",
 ]
+
+[features]
+default = ["helpers"]
+helpers = ["dep:witty-phrase-generator"]
```

Comment:
note: we could replace `witty-phrase-generator` with tempfile (native) and rand (web)